content/docs/1.5.0/terminology.md

```diff
@@ -9,11 +9,12 @@ weight: 4
 - [Backup target](#backup-target)
 - [Backup volume](#backup-volume)
 - [Block storage](#block-storage)
+- [CRD](#crd)
```

Comment:
This is another good example again to do the refactoring or a minor fix together with a new change. 

cc @longhorn/dev